src/main/java/org/opentripplanner/street/search/state/State.java

```diff
@@ -63,12 +65,21 @@ public State(Vertex vertex, StreetSearchRequest streetSearchRequest) {
     );
   }
 
-  public State(Vertex vertex, Instant startTime, StateData stateData, StreetSearchRequest request) {
+  public State(
+    @Nonnull Vertex vertex,
+    Instant startTime,
+    StateData stateData,
+    StreetSearchRequest request
+  ) {
```

Comment:
Same here, vertex is not the only nonnull parameter.

Reply:
I've added the annotations in b9aeef1751093f6f1eb9ac184cd829c90110a8ac